templates/CRM/Case/Form/Activity.tpl

```diff
@@ -271,18 +271,28 @@
 <div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="bottom"}</div>
 
   {if $action eq 1 or $action eq 2}
-  {*include custom data js file*}
-  {include file="CRM/common/customData.tpl"}
+    {*include custom data js file*}
+    {include file="CRM/common/customData.tpl"}
     {literal}
     <script type="text/javascript">
     CRM.$(function($) {
-    {/literal}
-    {if $customDataSubType}
-      CRM.buildCustomData( '{$customDataType}', {$customDataSubType} );
-      {else}
-      CRM.buildCustomData( '{$customDataType}' );
-    {/if}
-    {literal}
+      var doNotNotifyAssigneeFor = ["{/literal}{'","'|implode:$doNotNotifyAssigneeFor}{literal}"];
```

Comment:
I think best practice is to use `json_encode` whenever assigning smarty variables to js vars. E.g. 

    var doNotNotifyAssigneeFor = {/literal}{$doNotNotifyAssigneeFor|@json_encode}{literal};

I know there's lots of examples in tpl files where we don't do that but I've fixed quite a few bugs caused by improper escaping and edge cases that were solved by using `json_encode`.

Reply:
I've updated and tested the above json_encode assignment and it works fine 👍